pyomo/repn/plugins/parameterized_standard_form.py

```diff
@@ -224,6 +224,64 @@ def todense(self):
 
         return dense
 
+    def tocsr(self):
```

Comment:
We ought to follow scipy's lead and only implement a single method that `tocsr` and `tocsc` both use (instead of virtually duplicate code)